data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -444,7 +374,7 @@ private void fetchAllTokTableOrCol(ASTNode nodeSelExpr, String functionString, L
 				}
 				String columnFullName = (tableName!=null ? tableName+"." : "") + colName;
 
-				System.out.println("Input column(infunction) = " + colName);
+				LOGGER.info("Input column(infunction) = " + colName);
 				columnList.addToList("inputcol(infunction)->" + colName);
 				inColumnList.add(colName);
 				functionString += ";inputcolumn->" + colName;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "functionString". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1226)